package.json

```diff
@@ -2,7 +2,7 @@
   "name": "soi-portal",
   "description": "Substainable Ocean Initiative Portal",
   "dependencies": {
-    "bower": "1.8.0",
+    "bower": "^1.8.0",
```

Comment:
```suggestion
    "bower": "1.8.0",
```

Reply:
We try to not auto update packages as sometimes the new ones will have unexpected results.